Pod/Classes/Cards/CardType.swift

```diff
@@ -42,23 +42,63 @@ public protocol CardType {
+     
+     - important: When creating custom card types, you should make sure, that there are no conflicts in the Issuer Identification Numbers you provide. For example, using [309] to detect a Diners Club card and using [3096] to detect a JCB card will not cause issues as IINs are parsed with the highest numbers first, i.e. the numbers that provide the most context possible, which will return a JCB card in this case. However, no two card types should provide the exact same number (like [309] to detect both a Diners Club card and a JCB card)!
      
      - returns: A set of numbers which, when being found in the first digits of the card number, indicate the card issuer.
      */
     var identifyingDigits: Set<Int> { get }
 
+    /**
+     Validates the card verification code.
+     
+     - important: This validation does not mean that the CVC will be accepted by the card issuer! In order to validate the card with the card issuer, you should opt for the integration of a [Payment Gateway](https://en.wikipedia.org/wiki/Payment_gateway) to confirm the authenticity of the entered CVC.
+     
+     - parameter cvc: The card verification code as indicated on the payment card.
+     
+     - returns: The validation result for the CVC, taking the current card type into account, as different card issuers can provide CVCs in different formats.
+     */
     func validateCVC(cvc: CVC) -> CardValidationResult
 
+    /**
+     Validates the card number.
+     
+     - important: This validation incorporates the validation of the card number length, a test for the [Luhn algorithm](https://en.wikipedia.org/wiki/Luhn_algorithm) and a check, whether or not the card number only contains numeric digits. Within *Caishen*, no validation with a card issuer takes place and you should opt for the integration of a [Payment Gateway](https://en.wikipedia.org/wiki/Payment_gateway) to confirm the authenticity of the entered card number.
+     
+     - parameter number: The card number.
+     
+     - returns: `CardValidationResult.Valid` if the card has been validated or:
+        - `.NumberDoesNotMatchType`:  The card number's Issuer Identification Number does not match `self`.
+        - `.NumberIncomplete`:        The card number is incomplete and one or more digits are missing.
+        - `.NumberIsNotNumeric`:      The card number contains invalid characters.
+        - `.LuhnTestFailed`:          The Luhn algorithm test failed.
+        - `.UnknownType`:             The card number is of an unknown card type.
+        - `.NumberTooLong`:           The card number contains more digits than required.
+     */
     func validateNumber(number: Number) -> CardValidationResult
 
+    /**
+     Validates the card's expiry date, checking whether the card has expired or not.
+     
+     - parameter expiry: The card's expiry.
+     
+     - returns: `CardValidationResult.Valid` if the card has not expired or:
```

Comment:
Same here. I'm ok with just pointing out that it returns the result of the parse, and let the user evaluate the documentation for these enums specifically. I don't think Apple or any other framework includes the result enum parsing as well.